Scripts/AppCenterIdentity/build-appcenteridentity-ios-framework.sh

```diff
@@ -1,16 +1,8 @@
 #!/bin/sh
 
-source $(dirname "$0")/../build-ios-framework.sh AppCenterIdentity
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
 
-# Add MSAL.
-if [ -z $(otool -L "${INSTALL_DIR}/${FMK_NAME}" | grep 'libMSAL') ]; then
-  if [ -z "$MS_ARM64E_XCODE_PATH" ] || [ ! -d "$MS_ARM64E_XCODE_PATH" ]; then
-    echo "Use legacy Xcode and link MSAL via libtool."
-    libtool -static  "${INSTALL_DIR}/${FMK_NAME}" "${SRCROOT}/../Vendor/iOS/MSAL/MSAL.framework/MSAL" -o "${INSTALL_DIR}/${FMK_NAME}"
-  else
-    echo "Use arm64e Xcode and link MSAL via libtool."
-    env DEVELOPER_DIR="$MS_ARM64E_XCODE_PATH" /usr/bin/libtool -static  "${INSTALL_DIR}/${FMK_NAME}" "${SRCROOT}/../Vendor/iOS/MSAL/MSAL.framework/MSAL" -o "${INSTALL_DIR}/${FMK_NAME}"
-  fi
-fi
+source $(dirname "$0")/../build-ios-framework.sh AppCenterIdentity
```

Comment:
We don't need to merge MSAL lib anymore?

Reply:
This is a project dependency. The binary will contain MSAL binary.